dev-app/routes/components/pill/properties/index.html

```diff
@@ -57,6 +57,33 @@
 
         <c-divider></c-divider>
 
+        <c-divider></c-divider>
+
+        <div>
+            <l-stack spacing="var(--s-3)">
+                <c-h3>Font-Family, Font &amp; Link color</c-h3>
```

Comment:
we've been using the non capitol version here so in the docs the h3 has the name of the attribute like you'd use it in the code. So if we can change 'Font-Family' to just 'font-family' that would be perfect.
font-family, link-color, &amp; text-color